content/user-research/analysing-user-research/affinity.md

```diff
@@ -9,4 +9,4 @@ reverse: false
 imageAlt: "A wall with a lot of post-it notes in different colours."
 ---
 
-‘We did affinity mapping in Alpha; other teams do it earlier in Discovery. It’s what we did after we spoke to the users. We used it to gather the user insights to find high-level topics and headings. Those fed into design hypotheses. Going analogue like this gave us good visibility. It’s hands on. If we did it digitally straight away the team might not have had the level of comfort to all get involved.’ — Delivery manager
+‘We did affinity mapping in Alpha; other teams do it earlier in Discovery. It’s what we did after we spoke to the users. We used it to gather the user insights to find high-level topics and headings. Those fed into design hypotheses. Going analogue like this gave us good visibility. It’s hands on. If we did it digitally straightaway the team might not have had the level of comfort to all get involved.’ — Delivery manager
```

Comment:
This is an attempt to _revert_ what's currently on prod. It should be the same as what's on develop: 'straight away'. @meaghan-n correct?